css/style.css

```diff
@@ -411,76 +728,239 @@
     padding-right: 35px;
   }
 
+  .data__figures {
+    font-weight: 600;
+  }
+
   .test-btn {
     padding-right: 0;
   }
 
   .testdrive {
     margin-right: 60px;
   }
-}
-
-@media (min-width: 900px) and (max-width: 1023px) {
-  .menubar__link {
-    display: none;
-  }
 
-  .tesla-bgd {
-    top: -35px;
-    left: 0;
-    width: 100%;
+  .info__img {
     height: auto;
-    max-width: 1023px;
-  }
-
-  .tesla-car {
-    padding: 54px 0 20px 0;
     width: 100%;
+    max-width: 1199px;
   }
 
-  .content__data-car {
-    padding: 0 calc(321px - 2 * (321px - (321 * 100% / 1440))) 50px;
-    flex-direction: column;
+  .wrap-information, .wrap {
+    height: 747px;
   }
 
-  .content__facts {
-    padding-bottom: 65px;
+  .red-first {
+    width: 400px;
+    height: 100%;
+    padding: 37px 47px 0 45px;
   }
 
-  .data__speed-block {
-    padding-right: 104px;
+  .description__title {
+    font-size: 25px;
   }
 
-  .data__speed-icon {
-    width: 65px;
-    height: 50px;
+  .drive {
+    padding-bottom: 60px;
   }
 
-  .content__road-icon {
-    width: 43px;
-    height: 71px;
+  .img-inch {
+    width: 30px;
+    height: 47px;
   }
 
-  .data__dimention {
-    align-items: center;
+  .data__figure {
+    font-size: 32px;
   }
 
-  .data__figures {
-    font-size: 50px;
+  .data__ins {
+    font-size: 19px;
   }
 
-  .data__specifications {
-    display: block;
+  .img-wifi {
+    width: 38px;
+    height: 28px;
   }
 
-  .testdrive__img {
-    display: none;
+  .data__wireless-update {
+    font-size: 18px;
   }
 
-  .testdrive__text {
-    position: relative;
-    font-size: 16px;
-    color: #e51937;
+  .description__bag {
+    padding-bottom: 70px;
+  }
+
+  .img-cap {
+    width: 30px;
+    height: 44px; 
+  }
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)